packages/core/package.json

```diff
@@ -54,9 +54,11 @@
     "lodash.omit": "^4.5.0",
     "lodash.tonumber": "^4.0.3",
     "react-native-deck-swiper": "^2.0.12",
+    "react-native-gesture-handler": "^2.9.0",
```

Comment:
The recommended version for these packages for expo 46 are:
    "react-native-gesture-handler": "~2.5.0",
    "react-native-reanimated": "~2.9.1"

The recommended versions for expo 47 are:
    "react-native-gesture-handler": "~2.8.0",
    "react-native-reanimated": "~2.12.0"

Is it possible to use these sets with this component? Otherwise I worry we'll run into conflicts when these apps need to be built in expo 46/47

Reply:
I remember trying these versions initially and running into issues on draftbit. However, just tried them now, and everything works. In fixing other issues, these were probably fixed as well.